### DISCUSSION TITLE
Nie działa!? Po aktualizacji przestał działać Nominatim i OpenCage.

### DISCUSSION THREAD

**Discussion by rbtm** [YetiForceCompany/YetiForceCRM#16373] [General]
Dzień dobry, zrobiłem dziś aktualizację i przestały działać obie wyszukiwarki adresów jak w temacie. Przed aktualizacją a po wpisaniu fragmentu adresu uruchamiało się wyszukiwanie a obecnie nie mogę wymusić tego w żaden sposób?! Albo takiego sposobu nie znam, albo zostało coś popsute :-(

**Comment by mariuszkrzaczkowski** [comment]
Zgłoszenia błędów proszę pisać w dziale issues z kompletem wymaganych  informacji
